### ISSUE TITLE
Outlook Message binding returns an error on Functions v2 Runtime and JavaScript.

### ISSUE THREAD

**Issue by FinVamp1** [Azure/azure-functions-microsoftgraph-extension#89]
**Error Message**

```
Microsoft.Azure.WebJobs.Host.FunctionInvocationException : Exception while executing function: Functions.NotifyBoxUsersOfCollabotorChange ---> System.ArgumentException : Object serialized to Bytes. JObject instance expected.
   at Newtonsoft.Json.Linq.JObject.FromObject(Object o,JsonSerializer jsonSerializer)
   at Newtonsoft.Json.Linq.JObject.FromObject(Object o)
   at Microsoft.Azure.WebJobs.Extensions.MicrosoftGraph.Config.Converters.OutlookGenericsConverter`1.Convert(T input)
```

**Repro Steps*

https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-microsoft-graph#outlook-output  

**Expected Result**
The binding should send messages

**Actual Result**
The error is returned as above.

**Comment by ConnorMcMahon** [comment]
There is a fix in PR #80.

**Comment by ahuggins-nhs** [comment]
This issue persists in 1.0.0-beta6.
